functorch/setup.py

```diff
@@ -1,149 +0,0 @@
-# Copyright (c) Facebook, Inc. and its affiliates.
-# All rights reserved.
-#
-# This source code is licensed under the BSD-style license found in the
-# LICENSE file in the root directory of this source tree.
-
-import distutils.command.clean
-import shutil
```

Comment:
We've got a UX problem where the way dynamo installs functorch no longer works (https://github.com/pytorch/torchdynamo/blob/d88a0ac0e4a799617e59a05fec79d329781f3ea7/Makefile#L54-L55 ) due to setup.py not being legit anymore. I'm not sure how to resolve this: maybe we should just have a trivial setup.py?

Reply:
There are at least three users for this: https://github.com/search?q=subdirectory%3Dfunctorch&type=code . To err on the safe side, we should have a dummy setup.py that does invoke setuptools